integrations/azure-devops/azure_devops/http/http_client.py

```diff
@@ -0,0 +1,150 @@
+import httpx
+from httpx import Timeout, BasicAuth, Response
+from typing import Any, AsyncGenerator, Optional
+from port_ocean.utils import http_async_client
+from loguru import logger
+
+PAGE_SIZE = 50
+REQUEST_TIMEOUT_SECONDS = 30
+CONTINUATION_TOKEN_HEADER = "x-ms-continuationtoken"
+ALLOW_REDIRECTS = True
+
+
+class HTTPBaseClient:
+    def __init__(self, personal_access_token: str) -> None:
+        self._async_client = http_async_client
+        # Username isn't required in basic auth to Azure Devops
+        self._auth = BasicAuth("", personal_access_token)
+        self._timeout = Timeout(REQUEST_TIMEOUT_SECONDS)
+        self._follow_redirects = ALLOW_REDIRECTS
```

Comment:
why is this needed?

Reply:
The Azure devops rest api backend uses redirects, Without this requests fail.